workflow/scripts/ice-floe-tracker.jl

```diff
@@ -79,11 +79,11 @@ function main(args)
     # delete log option from command_args so it doesn't get passed to command_func
     delete!(command_args, :log)
 
-    logger = setuplogger(logoption, command)
+    # logger = setuplogger(logoption, command)
 
-    with_logger(logger) do
-        @time command_func(; command_args...)
-    end
+    # with_logger(logger) do
+         @time command_func(; command_args...)
+    # end
```

Comment:
Why comment out the logger? If it's not needed, some deps (LoggingExtras) could also be dropped.

Reply:
I think it was the issue with the logger trying to write in the read-only container. I can retest it and see if it will work now that we've updated some other code. It's possible that we have the right bindings now and it might work.

Reply:
I was able to add a binding so that it will write the logs to `workflow/report`!